### DISCUSSION TITLE
Currently testing soccer markets and different leagues' endpoints, and have had issues with some upcoming matches.

### DISCUSSION THREAD

**Discussion by Braziliandre** [cloudbet/docs#89] [Q&A]
So I've been trying to test basically all soccer leagues and the market endpoints, and I've had some issues with some of the games, even though they are listed on the website. For example, for the match below, the endpoint API call failed... 

Soccer, Bulgaria Parva Liga
08-18 13:15
Hebar vs Botev Plovdiv
Asian Handicap: Away 0.75 [1.45]

How far out can the API reach a match for? Maybe some leagues aren't quite integrated yet? Any insight would be helpful here! Also would love to hear any suggestions on an effective way to test the various endpoints to make sure they are working before employing a betting strategy for example. thanks!

**Comment by kgravenreuth** [comment]
Hi @Braziliandre ,

Could you please post a sample curl request + the error response status and body so that we can investigate further? 

In general, any events available on the cloudbet.com website should also be available on the Cloudbet Feed API, while there can be some restrictions such as Geo restrictions or Rate Limits for betting as mentioned in the [Cloudbet Trading API docs](https://www.cloudbet.com/api/?urls.primaryName=Trading).

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet
